pegasus/sites.v3/code.org/views/elementary_school_curricula_video.haml

```diff
@@ -0,0 +1,48 @@
+- video_middle_curricula_index = 5
+
+- video_middle_curricula_tags = [hoc_s(:module_grade_k_5), hoc_s(:module_grade_3_5), hoc_s(:module_grade_3_12), hoc_s(:module_grade_3_12), hoc_s(:module_grade_k_12)]
+
+- video_middle_curricula_title = [hoc_s(:curriculum_name_csf), hoc_s(:curriculum_name_csc), hoc_s(:curriculum_name_artificial_intelligence), hoc_s(:curriculum_name_maker), hoc_s(:hour_of_code)]
+
+- video_middle_curricula_desc = [hoc_s(:csf_page_top_desc), hoc_s(:csc_page_top_desc), hoc_s(:ai_hero_desc), hoc_s(:curriculum_desc_maker), hoc_s(:hoc_homepage_what_is_hoc_subhead)]
+
+- video_id = ["cs_connections", "ai_intelligence", "courses_maker"]
+
+- video_code = ["", "GDcKJ1v2W1s", "dWRnCXbUDgA", "ZoqMiFKspAA"]
+
+
+- video_middle_curricula_index.times do |index|
+  .action-block.action-block--two-col.video-101__text
+    - if index === 1|| index === 2 || index === 3
+      %figure.video-101__video.col-50
+        = view :display_video_thumbnail, id: video_id[index], video_code: video_code[index], play_button: 'center', letterbox: 'false'
+    - if index === 4
+      .video-101__image
+        %img{src: "/images/hoc2022-banner.png", style: "width: 100%" }
+    - if index === 0
+      .video-101__image
+        %img{src: "/images/curriculum/CS-Fundamentals.jpg", style: "width: 100%"}

```

Comment:
This should be showing this video here instead of an image ([Figma comment](https://www.figma.com/file/xPFXkCz4m96FKP3RJWQ1i4?node-id=1677:625&mode=design#530257786)): https://github.com/code-dot-org/code-dot-org/blob/dd2c5a2a85afc2af6e2e8f70a6ebdb03204ec3bb/pegasus/sites.v3/code.org/public/educate/csf.haml#L25